Makefile

```diff
@@ -9,6 +9,8 @@ VERSION_TAG ?= v$(VERSION)
 # Set this when when there is a new patch release in the channel.
 REPLACES_VERSION ?= $(shell cat REPLACES_VERSION)
 
+GATEKEEPER_VERSION ?= v3.14.0
```

Comment:
I think this is fine for now since it's directly into the `release-3.14` branch, but for `main` I'm thinking we could have a different process where we create a `GATEKEEPER_VERSION` if there's skew between the operator version and Gatekeeper version. I'm thinking this could look like:
```
GATEKEEPER_VERSION ?= $(shell cat GATEKEEPER_VERSION || cat VERSION)
```
I feel like this a little bit more maintainable/discoverable.

Reply:
lovely Thanks!